### ISSUE TITLE
Script/XMLRPC Call shows wrong Type/FW Version

### ISSUE THREAD

**Issue by DanielWeeber** [eq-3/occu#91]
see above

`string stderr;
string stdout;
system.Exec("/bin/sh -c '" # 'echo "load tclrpc.so; puts [xmlrpc http://127.0.0.1:2001/ listBidcosInterfaces ]" | tclsh' # "'", &stdout, &stderr);
WriteLine(stdout);`

shows

`{ADDRESS PEQ1931070 CONNECTED 1 DEFAULT 1 DESCRIPTION {} DUTY_CYCLE 9 FIRMWARE_VERSION 3.4.8 TYPE CCU2}`

I'm using a CCU3 with FW version 3.41.11

**Comment by HMside** [comment]
Hier wird nur die CoPro FW Version und nicht die CCU FW ausgegeben.

**Comment by foxriver76** [comment]
@HMside jedoch sollte TYPE trotzallem CCU3 ausgeben oder liege ich damit falsch?

**Comment by jens-maus** [comment]
@foxriver76 Ja, damit liegst du falsch. Der String bleibt auch bei einer CCU3 bei "CCU2"...

**Comment by foxriver76** [comment]
@jens-maus Ergibt für mich keinen Sinn, dass eine CCU3 sich an dieser Stelle als CCU2 ausgeben sollte. Auf mich wirkt es nach einem Fehler, allerdings lasse ich mich gerne eines besseren belehren mit einer logischen Begründung. :-)

**Comment by jens-maus** [comment]
@foxriver76 So ist das halt im Moment. Ist ohnehin ne Legacy ausgäbe das ganz. Eine RaspberryMatic schreibt da auch nicht "RaspberryMatic" rein. Wenn man rausfinden möchte ob es sich um eine CCU3, CCU2 oder CCU1 handelt muss man anders rangehen und eben z.B. das `/VERSION` file auslesen um die Firmwareversion zu bekommen und wenn die Version 3.x ist ist es eine CCU3 und wenn 2.x ist es eine CCU2, usw...
